libtelemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java

```diff
@@ -16,12 +16,13 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
 
-import okhttp3.Callback;
+import okhttp3.Response;
 
 import static com.mapbox.services.android.telemetry.EventReceiver.EVENT_RECEIVER_INTENT;
 
-public class MapboxTelemetry implements FullQueueCallback, EventCallback {
+public class MapboxTelemetry implements FullQueueCallback, EventCallback, TelemetryListener {
```

Comment:
This should be tackled differently 👀 https://github.com/mapbox/mapbox-events-android/pull/38#discussion_r161520797